src/crewai/agents/executor.py

```diff
@@ -185,6 +198,55 @@ def _iter_next_step(
             yield AgentStep(action=output, observation=observation)
             return
 
+        except Exception as e:
+            if LLMContextLengthExceededException(str(e))._is_context_limit_error(
+                str(e)
+            ):
+                if "context_length_exceeded" in str(e):
+                    self._logger.log(
+                        "debug",
+                        "Context length exceeded. Asking user if they want to use summarize prompt to fit, this will reduce context length.",
+                        color="yellow",
+                    )
+                    user_choice = click.confirm(
+                        "Context length exceeded. Do you want to summarize the text to fit models context window?"
```

Comment:
_Question_: I imagine that at the end is missing something like a (y). If the user presses enter without anything or just types anything, how it work?